### ISSUE TITLE
Pattern return english error message. Other language set

### ISSUE THREAD

**Issue by HawkonDK** [jquery-validation/jquery-validation#2065]
Pattern return "Invalid format" in English even if you have chose another language

**Comment by Arkni** [comment]
Not all languages files have the translation for `pattern` error message. We will need to get people to contribute the needed translations.

If someone is interested in adding a translation for `pattern` error message, I will be happy to merge their PR.

Thanks!

**Comment by HawkonDK** [comment]
well I can do it for the Danish(da-dk) version

**Comment by Arkni** [comment]
That would be good. Thanks!

**Comment by HawkonDK** [comment]
what to write inside the messages_da.js to change the error messages for pattern?

**Comment by Arkni** [comment]
The format is: `methodName: "translation"`. In this case, you just need to add one line:

```js
pattern: "the translation in danish"
```

Thanks!

**Comment by HawkonDK** [comment]
Oh there is a error in the franch version at line 53 missing  `$.validator.format()`

**Comment by Arkni** [comment]
I think you mean line 46. Yes, that's definitely a bug in translation. Please open a PR to fix that to.

Just for your information, we follow a convention for the commit message. See point 5 of [`Contributing code`](https://github.com/jquery-validation/jquery-validation/blob/fa73b79b1d714356bbf365b77110b8c4af7e8491/CONTRIBUTING.md#contributing-code) in CONTRIBUTING.md:
```
5. Describe the change in your commit message and reference the ticket, like
   this: "Demos: Fixed delegate bug for dynamic-totals demo. Fixes #51".
   If you're adding a new localization file, use something like this:
   "Localization: Added croatian (HR) localization"
```

Thanks!
